src/main/java/org/mozilla/jss/pkcs11/PK11Signature.java

```diff
@@ -311,10 +315,6 @@ public boolean engineVerify(byte[] sigBytes)
 		assert(algorithm!=null);
 		assert(key!=null);
 
-		if(sigBytes==null) {
-			return false;
-		}
```

Comment:
I think it's actually possible for `sigBytes` to be `null` here in case the assertion is not enabled, so we probably should keep this code.

Reply:
I'm preparing another PR to remove use of `assert`, as you say you can disable them. They're also poor control flow as nobody is expecting to handle 'Error`.

Reply:
Java assertions are actually [disabled by default](https://docs.oracle.com/cd/E19683-01/806-7930/assert-4/index.html). I think it's meant to be enabled during development to complement testing (because it may not be possible to test all scenarios). But considering we never explicitly enable assertions, and the assertions we have so far might not have been used properly, we probably can just remove them or replace them with regular code.

Reply:
#854 <-Like so :smile: